packages/fe-mockserver-core/src/mockdata/functionBasedMockData.ts

```diff
@@ -161,7 +161,32 @@ export class FunctionBasedMockData extends FileBasedMockData {
             getEmptyObject: super.getEmptyObject.bind(this),
             getDefaultElement: super.getDefaultElement.bind(this),
             getParentEntityInterface: super.getParentEntityInterface.bind(this),
-            getEntityInterface: super.getEntityInterface.bind(this),
+            getEntityInterface: async (entitySetName: string, serviceNameOrAlias?: string) => {
+                const rawInterface = await super.getEntityInterface.call(this, entitySetName, serviceNameOrAlias);
+
+                if (!rawInterface) {
+                    return rawInterface;
+                }
+
+                // If this is a cross-service call, enhance with smart updateEntry behavior
+                if (serviceNameOrAlias) {
```

Comment:
What's this part for?

Reply:
when i updated a entry this way it did not update for example only the description but the whole entity
so, when i gave `{"description": "TETS"}` as a parameter, the logic updated the description and because i did not mention all the other parameter they got nulled
as this logic is not in the updateEntry but in the base.updateEntry i added the same logic there
you know a better way?

Reply:
Oh, i see, so this is more or less an existing limitation of the getEntityInterface that you're removing here :)

Reply:
yep looks good